src/components/themeProvider/index.tsx

```diff
@@ -1,16 +1,17 @@
 import React, { FC, PropsWithChildren } from 'react';
 import { ChakraProvider, CSSReset } from '@chakra-ui/react';
 
-import { autoScout24Theme, motoScout24Theme } from '../../theme';
+import { autoScout24Theme, motoScout24Theme, sharedTheme } from '../../theme';
 
 export type Props = {
   // Theme to use
-  theme: 'as24' | 'ms24';
+  theme: 'as24' | 'ms24' | 'base';
```

Comment:
What's the motivation behind creating a separate theme for shared values? If we go this route can we make sure it's not exposed to the consumer?